StyleCop.Analyzers/StyleCop.Analyzers/NamingRules/SA1300ElementMustBeginWithUpperCaseLetter.cs

```diff
@@ -29,6 +30,9 @@ namespace StyleCop.Analyzers.NamingRules
     /// class. A <c>NativeMethods</c> class is any class which contains a name ending in <c>NativeMethods</c>, and is
     /// intended as a placeholder for Win32 or COM wrappers. StyleCop will ignore this violation if the item is placed
     /// within a <c>NativeMethods</c> class.</para>
+    ///
+    /// <para>For namespace components that begin with a small letter, due to branding issues or other reasons, add the
+    /// term to the <c>allowedNamespaceComponentTerms</c> list.</para>
```

Comment:
```suggestion
    /// term to the <see href="https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/ed1159193cd1642ee149674eb8c19f4bffa27670/documentation/Configuration.md#namespace-components"><c>allowedNamespaceComponentTerms</c> list</see>.</para>
```

Reply:
📝 These hyperlinks and styles actually work in the IDE now, aside from the fact that `<remarks>` are currently hidden. But they _will_ work eventually 😆